docs/rst/introduction/middleware.rst

```diff
@@ -0,0 +1,78 @@
+.. _vulcanexus_middleware:
+
+Vulcanexus MiddleWare
+=====================
+
+Vulcanexus supports *eProsima Fast DDS* as the official middleware.
+*Fast DDS* provides many features of interest to the development of robotics applications, providing also custom solutions to well-known problems as, for instance, discovery issues for large deployments.
+This section provides an introduction to *Fast DDS* and its main advantages.
+More information can be found in `Fast DDS documentation <https://fast-dds.docs.eprosima.com/en/latest/>`_.
+
+.. _vulcanexus_fastdds:
+
+Fast DDS
+--------
+
+*eProsima Fast DDS* is a C++ implementation of the `DDS (Data Distribution Service) Specification <https://www.omg.org/spec/DDS/About-DDS/>`__, a protocol defined by the `Object Management Group (OMG) <https://www.omg.org/>`__.
+The *eProsima Fast DDS* library provides both an Application Programming Interface (API) and a communication protocol that deploy a Data-Centric Publisher-Subscriber (DCPS) model, with the purpose of establishing efficient and reliable information distribution among Real-Time Systems.
+*eProsima Fast DDS* is predictable, scalable, flexible, and efficient in resource handling.
+For meeting these requirements, it makes use of typed interfaces and hinges on a many-to-many distributed network paradigm that neatly allows separation of the publisher and subscriber sides of the communication.
+
+Main Features
```

Comment:
I think it'd be neat if we added some links to the already existing Vulcanexus tutorials and docs in some of these bullet points.